### ISSUE TITLE
Concept Prompting

### ISSUE THREAD

**Issue by InternalMegaT** [TheLastBen/fast-stable-diffusion#872]
Is there a way to add a concept prompt to the model your training? Like if you want to specifically use your trained model you have to type "jhjr style" in your prompt

**Comment by TheLastBen** [comment]
Yes, it already exists, it's the images filename, the unique identifier can be used to summon the concept trained on
